docs/API.md

```diff
@@ -445,6 +445,10 @@ Options can be configured as a parameter to the [init()](./interfaces/tracer.htm
 | -               | `DD_TRACE_DISABLED_PLUGINS`       | -              | A comma-separated string of integration names automatically disabled when tracer is initialized. Environment variable only e.g. `DD_TRACE_DISABLED_PLUGINS=express,dns`. |
 | logLevel        | `DD_TRACE_LOG_LEVEL`              | `debug`        | A string for the minimum log level for the tracer to use when debug logging is enabled, e.g. `'error'`, `'debug'`. |
 | startupLogs     | `DD_TRACE_STARTUP_LOGS`           | `true`         | Enable tracer startup configuration and diagnostic log. |
+<!---
+TODO(simon-id): put this in the public API (private beta for now)
+| appsec.beta     | `DD_APPSEC_BETA`                  | `false`        | Enable the beta AppSec module. |
```

Comment:
Usually this would be `experimental.appsec` with the current configuration.

Reply:
I removed the documention entry for now, as it's supposed to be a private beta feature, and it will have a different name once GA comes around